### ISSUE TITLE
load模型进行test 和训练时候test的结果存在较大差异问题：

### ISSUE THREAD

**Issue by JNUpython** [zhouhaoyi/Informer2020#220]
train test保存模型后；
load 已经保存同一个模型，再进行test存在结果不一致问题

这个过程进行随机种子设定：
def set_seed(seed=2021):
    logger.info("set seed: %s" % seed)
    random.seed(seed)
    np.random.seed(seed)
    torch.manual_seed(seed)
    if torch.cuda.is_available():
        torch.cuda.manual_seed(seed)

保存和加载模型用如下办法：
 # torch.save(exp.model.state_dict(), 'testcheckpoint.pth')
    torch.save(exp.model, 'net_model.pkl')

 # exp.model.load_state_dict(torch.load("testcheckpoint.pth", map_location=torch.device('cpu')))
    exp.model = torch.load('net_model.pkl')
